draft-pauly-intarea-proxy-config-pvd.md

```diff
@@ -275,14 +309,37 @@ proxies, can only be safely used when fetched over a secure TLS-protected connec
 and the client has validated that that the hostname of the proxy, the identifier of
 the PvD, and the validated hostname identity on the certificate all match.
 
-# IANA Considerations {#iana}
+# IANA Considerations
+
+## New PvD Additional Information key {#proxies-key-iana}
 
 This document registers a new key in the "Additional Information PvD Keys" registry.
 
 JSON Key: proxies
 
-Description: Array of proxy URIs associated with this PvD
+Description: Array of proxy dictionaries associated with this PvD
+
+Type: Array of dictionaries
+
+Example: [ { "protocol": "connect", "uri":"https://proxy.example.com" } ]
+
+## New PvD Proxy Information Registry {#proxy-info-iana}
+
+IANA is requested to create a new registry "Proxy Information PvD Keys", within the "Provisioning Domains (PvDs)" registry page.
+This new registry reserves JSON keys for use in sub-dictionaries under the `proxies` key.
+The initial contents of this registry are given in {{proxy-enumeration}}.
+
+The status of a key as mandatory or optional is intentionally not denoted in the table to allow for
+flexibility in future use cases. Any new assignments of keys will be considered as optional for the purpose of the mechanism described in this document.
+
+New assignments in the "Proxy Information PvD Keys" registry will be administered by IANA through Expert Review {{!RFC8126}}. Experts are
```

Comment:
Should we define an allowed character set for keys?

Reply:
Would that be something that would be inherited from the JSON string requirements? We didn't have any such requirement for the main registry spelled out I think.

Reply:
Technically JSON allows any possible character in keys. That said, some languages have additional restrictions (some C libraries will interpret a null-byte as the end of the key for example, or some high-level languages turn keys into methods but then have requirements on characters allowed in method names such as no spaces, ...). I realize that 8801 didn't constrain its set, but perhaps we can do better here? Wanting consistency makes sense though, in. that case SHOULD limit the characters might be enough

Reply:
Filed https://github.com/tfpauly/privacy-proxy/issues/246